presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/SuiteRun.java

```diff
@@ -153,28 +154,41 @@ public void run()
             }
 
             List<TestRunResult> testRunsResults = results.build();
-            boolean allSuccessful = testRunsResults.stream()
-                    .allMatch(TestRunResult::isSuccessful);
-
-            if (allSuccessful) {
-                log.info("Suite succeeded in %s:", nanosSince(suiteStartTime));
-                printTestRunsSummary(testRunsResults);
-                exit(0);
-            }
-
-            log.error("Suite failed in %s:", nanosSince(suiteStartTime));
-            printTestRunsSummary(testRunsResults);
-            exit(1);
+            printTestRunsSummary(suiteStartTime, suiteName, testRunsResults);
         }
 
-        private static void printTestRunsSummary(List<TestRunResult> results)
+        private static void printTestRunsSummary(long startTime, String suiteName, List<TestRunResult> results)
         {
             long failedRuns = results.stream()
                     .filter(TestRunResult::hasFailed)
                     .count();
 
-            log.info("Test runs summary (%d passed, %d failed): ", results.size() - failedRuns, failedRuns);
-            results.forEach(result -> log.info(" * '%s' %s in %s", result.getSuiteRun(), result.isSuccessful() ? "PASSED" : "FAILED", result.getDuration()));
+            if (failedRuns > 0) {
+                log.info("Suite %s failed in %s (%d passed, %d failed): ", suiteName, nanosSince(startTime), results.size() - failedRuns, failedRuns);
+            }
+            else {
+                log.info("Suite %s succeeded in %s: ", suiteName, nanosSince(startTime));
+            }
+
+            results.stream()
+                    .filter(TestRunResult::hasFailed)
+                    .forEach(Execution::printTestRunSummary);
+
+            results.stream()
```

Comment:
I am not convinced we need those. And even if we do,  I would put them before failed ones. So it is easy to scroll down to failed list of failed tests on GHA.